librz/include/rz_cons.h

```diff
@@ -496,6 +496,19 @@ typedef struct rz_cons_context_t {
 
 #define HUD_BUF_SIZE 512
 
+typedef enum {
+	MOUSE_NONE,
```

Comment:
i think some code still assumes that MOUSE_NONE == 0? if so, do `MOUSE_NONE = 0,` please

Reply:
```suggestion
	MOUSE_NONE = 0,
```